packages/@uppy/companion/src/config/grant.js

```diff
@@ -29,6 +29,12 @@ module.exports = () => {
       transport: 'session',
       scope: ['files.read.all', 'offline_access', 'User.Read'],
       callback: '/onedrive/callback'
+    },
+    zoom: {
+      transport: 'session',
+      authorize_url: 'https://zoom.us/oauth/authorize',
+      access_url: 'https://zoom.us/oauth/token',
+      callback: '/zoom/callback'
```

Comment:
we need to add the permission scopes that the API will be using kind of like [this one](https://github.com/transloadit/uppy/blob/e9651443c7eb1eca0de93d98b3d960d7738efd19/packages/%40uppy/companion/src/config/grant.js#L24)